README.md

```diff
@@ -1 +1,637 @@
-# spree-storefront-api-v2-js-sdk
+# Spree Storefront API SDK
+
+Node module used for integration front-end with Spree API V2
+---
+
+
+
+Сontents:
+
+- [Quick Start](#quick-start)
+- [Example](#example)
+- [Endpoints](#endpoints)
+  - [Authentication](#authentication)
+    - [getToken](#getToken)
+    - [refreshToken](#refreshToken)
+  - [Account](#account)
+    - [accountInfo](#accountInfo)
+    - [creditCardsList](#creditCardsList)
+    - [defaultCreditCard](#defaultCreditCard)
+    - [completedOrdersList](#completedOrdersList)
+    - [completedOrder](#completedOrder)
+
+
+<br/>
+
+## Quick Start
+
+1. Install SDK
+
+  `npm install spree-storefront-api-v2-js-sdk --save`
+
+<br/>
+
+2. Initialize `Client`
+
+  ```ts
+    import { Client } from 'spree-storefront-api-v2-js-sdk'
+
+    const client = Client({
+      host: 'http://localhost:3000'
+    })
+  ```
+
+<br/>
+
+## Endpoints
+
+Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides2.spreecommerce.org/api/v2)
+
+## Authentication
+
+### `getToken`
+
+Method `getToken` used for creates a Bearer token required to authorize calls API calls. [Read more](https://guides2.spreecommerce.org/api/v2/authentication)
+
+__parameters schema:__
+
+```ts
+  username: string
+  password: string
+  grant_type: string = 'password'
+```
+
+__success response schema:__
+
+```ts
+  access_token: string
+  token_type: string = 'Bearer'
+  expires_in: number
+  refresh_token: string
+  created_at: number
+```
+
+__failed response schema:__
+
+```ts
+  error: number
+```
+
+__Example:__
+
+```ts
+  import { Client } from 'spree-storefront-api-v2-js-sdk'
+
+  const client = Client({
+    host: 'http://localhost:3000'
+  })
+
+  try {
+    const token = await client.authentication.getToken({
```

Comment:
- There's `await` but no `async` visible.
- Can we instead use `then()`?

Reply:
I guess we should use new features. I can add `async` function definition if you want. But I think this example shouldn't be detailed, like you described above `I'd skip writing `import...` and `const client...` in every example.` we also cant skip  `async` function

Reply:
ok, let's leave as is for now